src/convertToObject.js

```diff
@@ -6,7 +6,11 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resut = {};
```

Comment:
The variable name 'resut' seems to be a typo. It would be better to use a more descriptive name that follows the naming conventions, such as 'styleObject' or 'cssObject', to enhance code readability.